Formula/wine.rb

```diff
@@ -134,6 +489,16 @@ def caveats
     s
   end
 
+  def post_install
+    # For fontconfig
+    ohai "Regenerating font cache, this may take a while"
+    system "#{libexec}/bin/fc-cache", "-frv"
+
+    # For net-snmp
+    (var/"db/net-snmp").mkpath
+    (var/"log").mkpath
+  end
+
   test do
     system "#{bin}/wine", "--version"
```

Comment:
Wine actually comes with some tools that we can test. The problem is, the first time they are run `wine` will want to create its config and display a GUI telling us to wait (and no flag to turn it off). So I don't know if the CI will like it :)